### PR TITLE
Update OLS reference to use road-core/service instead.

### PR DESCRIPTION
**Pull request by r0x0d**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,9 +60,8 @@ make install
 ### Getting a backend to manage your queries
 
 ShellAI depends on a backend service to interact with LLM providers (such as OpenAI, IBM WatsonX, etc...). For that
-purpose, we strongly recommend setting up [OpenShift Lightspeed
-Service](https://github.com/openshift/lightspeed-service). After you have everything in orer, the next steps should work
-without extra configuration.
+purpose, we strongly recommend setting up [Road Core Service](https://github.com/road-core/service). After you have
+everything in orer, the next steps should work without extra configuration.
```
Comment (major) [inline CONTRIBUTING.md:64]: `s/orer/order/`

**Review by major — CHANGES_REQUESTED** [review]
Just one little thing :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (samdoran) [inline CONTRIBUTING.md:64]:
```suggestion
everything in order, the next steps should work without extra configuration.
```

Comment (r0x0d) [reply CONTRIBUTING.md:64]:
```suggestion
everything in order, the next steps should work without extra configuration.
```

**Review by major — APPROVED** [review]

**Pull request merged**